Gruntfile.js

```diff
@@ -5,6 +5,11 @@ camelcase: ["error", {"properties": "never"}]
 */
 var testConfig = require('./build/test/config');
 
+var defaultWebpackConfig = {
```

Comment:
Thoughts on creating a "config factory" instead?

```js
const createWebpackConfig = (input, output) => ({
  devtool: false,
  mode: 'development',
  entry: path.resolve(__dirname, input),
  output: {
    filename: 'index.js',
    path: output
  }
})
```

Then we'd just call it like this:

```js
{
  webpack: {
    commonsUtils: createWebpackConfig('lib/commons/utils/index.js', 'tmp/commons/utils'),
    commonsForms: createWebpackConfig('lib/commons/forms/index.js', 'tmp/commons/forms'),
  }
}
```